test/axelarProvider.spec.ts

```diff
@@ -0,0 +1,352 @@
+import { expect } from 'chai'
+import { ethers } from 'ethers'
+import * as sinon from 'sinon'
+import { AxelarProvider } from '../src/crossChain/AxelarProvider'
+import { ERC20 } from '../src/abis/ERC20'
+import { Provider } from 'decentraland-connect'
+import { TransactionResponses } from '@0xsquid/sdk/dist/types'
+
+describe('AxelarProvider', () => {
+  let provider: AxelarProvider
+  let ethersRef: typeof ethers
+
+  beforeEach(() => {
+    provider = new AxelarProvider('https://testapi.squid.com')
+    ethersRef = ethers
+  })
+
+  afterEach(() => {
+    sinon.restore()
+    // @ts-ignore - Restore ethers reference
+    ethers = ethersRef
+  })
+
+  describe('Interface Creation', () => {
+    it('should create interface using ethers v6 syntax', () => {
+      // Mock ethers v6-like environment
+      const mockEthers = {
+        Interface: {
+          from: sinon.stub().returns({ mockInterface: true })
+        }
+      }
+      // @ts-ignore - Mocking ethers
+      ethers = mockEthers
+
+      // Call the function directly from the module
+      const result = (provider as any)._createInterface(ERC20)
+      expect(result).to.deep.equal({ mockInterface: true })
+      expect(mockEthers.Interface.from.called).to.be.true
+    })
```

Comment:
Would you mind modifying the tests so the environment is built according to our testing standards?
```suggestion
   describe('when on a ethers v6 environment', () => {
   beforeEach(() => {
         // Mock ethers v6-like environment
      const mockEthers = {
        Interface: {
          from: sinon.stub().returns({ mockInterface: true })
        }
      }
      // @ts-ignore - Mocking ethers
      ethers = mockEthers
   })

    it('should create interface using ethers v6 syntax', () => {
      // Call the function directly from the module
      const result = (provider as any)._createInterface(ERC20)
      expect(result).to.deep.equal({ mockInterface: true })
      expect(mockEthers.Interface.from.called).to.be.true
    })
  })
```

Reply:
Refactored on 646295f8a9ea2663689c2557bbab5ada0ddbb118